.github/workflows/python-linux.sh

```diff
@@ -25,7 +25,7 @@ sudo rm -rf keploy/
 
 # Update the global noise to ignore the Allow header.
 config_file="./keploy-config.yaml"
-sed -i 's/"header": {}/"header":{"Allow":[]}/' "$config_file"
+sed -i 's/header: {}/header: {"Allow":[]}/' "$config_file"
 
 # Wait for 5 seconds for it to complete
```

Comment:
why these changes?

Reply:
I made these changes to solve #1210 , I will be reverting this change and make a separate PR.